map.css

```diff
@@ -2,3 +2,79 @@
   height: 40vw;
   width: 100%;
 }
+
+#searchMap {
+  height: 40vw;
+  width: 100%;
+}
+
+#description {
+  font-family: Roboto;
+  font-size: 15px;
+  font-weight: 300;
+}
+
+#infowindow-content .title {
+  font-weight: bold;
+}
+
+#infowindow-content {
+  display: none;
+}
+
+#map #infowindow-content {
+  display: inline;
+}
+
+.pac-card {
+  margin: 10px 10px 0 0;
+  border-radius: 2px 0 0 2px;
+  box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  outline: none;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.3);
+  background-color: #fff;
+  font-family: Roboto;
+}
+
+#pac-container {
+  padding-bottom: 12px;
+  margin-right: 12px;
+}
+
+.pac-controls {
+  display: inline-block;
+  padding: 5px 11px;
+}
+
+.pac-controls label {
+  font-family: Roboto;
+  font-size: 13px;
+  font-weight: 300;
+}
+
+#pac-input {
+  background-color: #fff;
+  font-family: Roboto;
```

Comment:
let's put Roboto as font family for the whole container or whole body, no need to set it in every element

additionaly, we _always_ need to add a fallback, browser's font, for the time being before the outsourced fonts are loaded. so here it can be `font-family: 'Roboto', sans-serif;`
this is resulting in so called FOUT effect (flash of unstyled text), more visible for slower connections, you can read about it. we sometimes handle that by FOIT (flash of invisible text), and recently someone somewhere invented FOUX which is simulating the outsourced font until it is fully downloaded. (this is just a fun fact but good to know, please read about it in the meantime)

one last thing: i don't see if you are downloading this font anywhere??